### PR TITLE
chore: Update codeowners

### PR DESCRIPTION
**Pull request by sdankel**
## Description

- fixes the name for the swayex team (recently renamed)

## Checklist

- [ ] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
   - [ ] If my change requires substantial documentation changes, I have [requested support from the DevRel team](https://github.com/FuelLabs/devrel-requests/issues/new/choose)
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [ ] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [ ] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

**Review by Voxelot — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -34,3 +31,5 @@
 Cargo.lock @FuelLabs/leads
 /deployment/ @FuelLabs/leads
 
+# Everything else
+* @FuelLabs/leads
```
Comment (IGI-111) [inline .github/CODEOWNERS:35]: This does the opposite of what you think.

CODEOWNER rules are set to the last matching rule for every changed file. putting the catchall at the end means every file has to be reviewed by a team lead, which is not what we want.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-/docs/ @FuelLabs/tooling @FuelLabs/sway-compiler @FuelLabs/application-dev @FuelLabs/Devrel
+/docs/ @FuelLabs/tooling @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/Devrel
 
 # Sway files and standard library
-/examples/ @FuelLabs/sway-compiler @FuelLabs/application-dev 
-/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/application-dev
-/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/application-dev
+/examples/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
+/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
+/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
```
Comment (IGI-111) [inline .github/CODEOWNERS:7]: I don't think tooling should get signing rights on the standard library.

**Review by IGI-111 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (sdankel) [reply .github/CODEOWNERS:35]: Ah sorry, I thought that's what was happening, but you're right. I'll remove it.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
 # Documentation
 /docs/ @FuelLabs/tooling @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/Devrel
 
 # Sway files and standard library
-/examples/ @FuelLabs/sway-compiler @FuelLabs/swayex 
-/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/swayex
-/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/swayex
+/examples/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
+/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
+/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
```
Comment (sdankel) [inline .github/CODEOWNERS:7]:
```suggestion
/examples/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/swayex
/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/swayex
```
```suggestion
/examples/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
/sway-lib-core/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
/sway-lib-std/ @FuelLabs/sway-compiler @FuelLabs/swayex @FuelLabs/tooling
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (sdankel) [inline .github/CODEOWNERS:35]:
```suggestion
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,13 +1,10 @@
-# Everything else
-* @FuelLabs/leads
-
 # Documentation
```
Comment (sdankel) [inline .github/CODEOWNERS:1]:
```suggestion
# Everything else
* @FuelLabs/leads

# Documentation
```

**Comment by sdankel** [comment]
Looks like you fixed the swayex team name issue, so closing this.

**Pull request closed without merging**